packages/components/src/utils/toast.ts

```diff
@@ -34,4 +35,21 @@ export const toast = {
      * Called when an error occured, to inform the user.
      */
     error: popToast('error'),
+    /**
+     * Handles HTTPError instances specifically and displays them as error toasts.
+     */
+    async errorWithDetails(err: unknown, fallbackTitle: string) {
```

Comment:
Maybe `toast.httpError` is a better name. or `toast.fromError`.

Think it's mostly bad initial design, should probably have been:

```
toast({ level: "error", ...options });
```

then we could have used `toast.error` for this, but too late now without a bunch of breaking changes.

Reply:
how about ``toast.error({ checkHttpError: true, fallbackTitle: `Failed to delete ${description}` })``